static/app/components/replays/replayController.tsx

```diff
@@ -115,17 +118,23 @@ const ReplayControls = ({
         <Button
           data-test-id="replay-fullscreen"
           size="xsmall"
-          title={t('View in full screen')}
+          title={isFullscreen ? t('Exit full screen') : t('View in full screen')}
+          aria-label={isFullscreen ? t('Exit full screen') : t('View in full screen')}
           icon={<IconResize size="sm" />}
-          onClick={onFullscreen}
-          aria-label={t('View in full screen')}
+          priority={isFullscreen ? 'primary' : undefined}
+          onClick={isFullscreen ? exitFullscreen : onFullscreen}
         />
       </ButtonGrid>
-    </Column>
+    </React.Fragment>
   );
 };
 
-const Column = styled('div')`
+const Panel = styled(_Panel)<{isFullscreen: boolean}>`
```

Comment:
I would avoid using `_` prefix in this way as it has a special meaning w/ eslint to denote as unused (e.g. when destructuring or an arg that is not used). Maybe `Base-` as prefix?